pkg/operator/configobserver/cloudprovider/observe_cloudprovider_test.go

```diff
@@ -199,6 +199,16 @@ func TestObserveCloudProviderNames(t *testing.T) {
 		},
 		expected:           "gce",
 		cloudProviderCount: 1,
+	}, {
+		name: "IBM Cloud platform",
+		infrastructureStatus: configv1.InfrastructureStatus{
+			Platform: configv1.IBMCloudPlatformType,
+			PlatformStatus: &configv1.PlatformStatus{
+				Type: configv1.IBMCloudPlatformType,
+			},
+		},
+		expected: "external",
```

Comment:
I think this should be empty to align with NonePlatformType.

Reply:
This still comes back with "external" based on the changes in the cloudprovider package